batch/batch/front_end/front_end.py

```diff
@@ -1285,7 +1310,7 @@ async def ui_get_billing(request, userdata):
 async def _query_billing_projects(db, user=None, billing_project=None):
     args = []
 
-    where_conditions = []
+    where_conditions = ["billing_projects.status != 'deleted'"]
```

Comment:
I'm not sure, but you might have to put status in backticks. I can't remember if it's a special word.